### DISCUSSION TITLE
Contingency Planning

### DISCUSSION THREAD

**Discussion by psuet** [netbirdio/netbird#1264] [Q&A / Support]
Hi,

Im currently evaluating Netbird. Is there anywhere some documentation about Failure Scenarios of Netbird.

I would like to know what happens if the Management Service, Signal Service, Coturn or the identity provider fails. Which connections do work? Does connection establishment work (if no STUN/TURN needed)?

Tailscale for example has some documentation that would clarify my concerns: https://tailscale.com/kb/1091/what-happens-if-the-coordination-server-is-down/

**Comment by Fantu** [comment]
based on my tests in case of issue or unreachable netbird server/s only P2P connections established remain active (if the route used is not compromised)
obviously if the connection or something on the route is temporarily lost they also fail and it is not possible to re-establish them, and obviously it is not possible to establish new ones with peers turned on later or re-establish them if the peer is restarted
